components/rewards/RewardsCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  Card,
+  ContentContainer,
+  ContentText,
+  ContentText2,
+  StarIcon
+} from '../../styled/rewards';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class RewardCard extends React.Component {
+  render() {
+    return (
+      <Card>
+        <StarIcon>
+          <FontAwesome5 name="star" size={24} color={Colors.darkerGreen} />
+        </StarIcon>
+        <ContentContainer>
+          <ContentText>$5 reward</ContentText>
+          <ContentText2>1000 points</ContentText2>
```

Comment:
You don't need separate containers for these. Pull straight from BaseComponents.